syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -23,6 +28,50 @@ def __public_unfold(image, kernel_size, padding):
     def __private_unfold(image, kernel_size, padding):
         return image.unfold(kernel_size, padding)
 
+    @staticmethod
+    def convert_b2a(
+        shares: List[PointerTensor], players: List[AbstractWorker], ring_size: int = 2 * 32
+    ) -> ReplicatedSharingTensor:
+        """
+        Convert from binary (field 2) to arithmetic share
+
+        Args:
+            shares (List[PointerTensor]): Pointers to a share, since is replicated
+                two workers has it
+            players (List[AbstractWorker]): The workers involved in the protocol
+            ring_size (int): The new ring_size
+
+        Returns:
+            A ReplicatedSharingTensor
+
+        """
+
+        player0 = shares[0].location
+        player1 = shares[1].location
+
+        shape = shares[0].shape
+
+        """
+        Since we shared the seeds for the PSRZ at the beginning we know
+        that zero_share_p0 + zero_share_p1 + zero_share_p2 == 0
+        """
+        zero_share_p0 = gen_alpha_3of3(players[0], shape=shape, ring_size=ring_size).wrap()
+        zero_share_p1 = gen_alpha_3of3(players[1], shape=shape, ring_size=ring_size).wrap()
+        zero_share_p2 = gen_alpha_3of3(players[2], shape=shape, ring_size=ring_size).wrap()
+
+        # RST for the zero share
+        share_p0 = [zero_share_p0, zero_share_p1.copy().move(players[0])]
+        share_p1 = [zero_share_p1, zero_share_p2.copy().move(players[1])]
+        share_p2 = [zero_share_p2, zero_share_p0.copy().move(players[2])]
+
+        arithmetic_shares = dict(zip(players, (share_p0, share_p1, share_p2)))
+        arithmetic_shares[player0][0] += shares[0]
+        arithmetic_shares[player1][1] += shares[1]
+
+        shares = {key: tuple(val) for key, val in arithmetic_shares.items()}
+
+        return ReplicatedSharingTensor(shares=shares)
+
     @staticmethod
     def xor(
```

Comment:
I really think the `__xor__` method shoud be added to the ReplicatedSharingTensor and not to the FalconHelper